Policies/github-corporate-terms-of-service.md

```diff
@@ -83,13 +90,13 @@ In some situations, third parties' terms may apply to your use of GitHub. For ex
 
 If you are a government User or otherwise accessing or using any GitHub Service in a government capacity, this [Government Amendment to GitHub Terms of Service](/articles/amendment-to-github-terms-of-service-applicable-to-u-s-federal-government-users/) applies to you, and you agree to its provisions.
 
-If you have signed up for GitHub Business Cloud, the [Business Cloud Addendum](/articles/github-business-cloud-addendum/) applies to you, and you agree to its provisions.
+If you have signed up for GitHub Enterprise Cloud, the [Enterprise Cloud Addendum](/articles/github-enterprise-cloud-addendum/) applies to you, and you agree to its provisions.
 
 ### C. Acceptable Use
 **Short version:** *GitHub hosts a wide variety of collaborative projects from all over the world, and that collaboration only works when our users are able to work together in good faith. While using the service, you must follow this Acceptable Use Policy, which includes some restrictions on content you can post, conduct on the service, and other limitations. In short, be excellent to each other.*
 
 #### 1. Compliance with Laws and Regulations
-Your use of the Website and Service must not violate any applicable laws, including copyright or trademark laws, export control laws, or other laws in your jurisdiction. You are responsible for making sure that your and your Users' use of the Service is in compliance with laws and any applicable regulations.
+Your use of the Website and Service must not violate any applicable laws, including copyright or trademark laws, export control or sanctions laws, or other laws in your jurisdiction. You are responsible for making sure that your and your Users' use of the Service is in compliance with laws and any applicable regulations.
```

Comment:
Should this be “export control laws, sanctions, or other laws in your jurisdiction”?

```suggestion
Your use of the Website and Service must not violate any applicable laws, including copyright or trademark laws, export control laws, sanctions, or other laws in your jurisdiction. You are responsible for making sure that your and your Users' use of the Service is in compliance with laws and any applicable regulations.
```

Reply:
@j-f1 Thanks for your feedback! Those two references have been grouped together, since they both fall under a broad category of trade compliance laws, and we intended to highlight those two specific subcategories of laws with the reference. It's similar to how we referenced "copyright or  trademark laws" in that sentence, as well.

Reply:
The reason I proposed this change is that “sanctions laws” seemed like a strange phrasing and [Google ngrams](https://books.google.com/ngrams/graph?content=sanctions%2Csanctions+laws&year_start=1800&year_end=2000&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Csanctions%3B%2Cc0%3B.t1%3B%2Csanctions%20laws%3B%2Cc0) confirms that “sanctions laws” is much rarer than just “sanctions” alone.

Reply:
I see what you're getting at, but this section is about compliance with laws, so what we're referring to is more specific:
- export control laws
- sanctions laws 

We've just grouped that together to read `export controls or sanctions laws`.

Reply:
Ok, that makes more sense. Thanks for responding!